dataParser.go

```diff
@@ -78,6 +78,7 @@ func (dp *dataParser) getSerializedElasticBlockAndHeaderHash(
 		PrevHash:              hex.EncodeToString(header.GetPrevHash()),
 		SearchOrder:           computeBlockSearchOrder(header),
 		GasConsumed:           gasConsumptionData.GasConsumed,
+		GasRefunded:           gasConsumptionData.GasRefunded,
```

Comment:
do we need to have these changes also on the new indexer branch?

Reply:
I have already integrated it in PR #58 - might add the gas refunded there as well